UPGRADE.rst

```diff
@@ -55,50 +55,140 @@ Upgrading to v1.4.0
+``trusted_third_party_id_servers`` will be used as the account threepid
+delegate for email. This is to ensure compatibility with existing Synapse
+installs that set up external server handling for these tasks before v1.4.0. If
+``email.trust_identity_server_for_password_resets`` is ``true`` and no trusted
+identity server domains are configured, Synapse will throw an error.
 
-If ``email.trust_identity_server_for_password_resets`` is ``false`` or absent and a threepid
-type in ``account_threepid_delegates`` is not set to a domain, then Synapse will attempt to
-send password reset and registration messages for that type.
+If ``email.trust_identity_server_for_password_resets`` is ``false`` or absent
+and a threepid type in ``account_threepid_delegates`` is not set to a domain,
+then Synapse will attempt to send password reset and registration messages for
+that type.
 
 Email templates
 ---------------
 
-If you have configured a custom template directory with the ``email.template_dir`` option, be
-aware that there are new templates regarding registration. ``registration.html`` and
-``registration.txt`` have been added and contain the content that is sent to a client upon
-registering via an email address.
+If you have configured a custom template directory with the
+``email.template_dir`` option, be aware that there are new templates regarding
+registration and threepid management.
+
+``registration.html`` and ``registration.txt`` contain the content that is sent
+to a client upon registering via an email address.
+
+``registration_success.html`` and ``registration_failure.html`` are templates
+that will be shown to the user when they click the link in their registration
+email , either showing them a success or failure page (assuming a redirect URL
+is not configured).
 
-``registration_success.html`` and ``registration_failure.html`` are also new HTML templates
-that will be shown to the user when they click the link in their registration emai , either
-showing them a success or failure page (assuming a redirect URL is not configured).
+``add_threepid.html`` and  ``add_threepid.txt`` are templates sent to a client
+to add the threepid.
 
-Synapse will expect these files to exist inside the configured template directory. To view the
-default templates, see `synapse/res/templates
+``add_threepid_failure.html`` and ``add_threepid_success.html`` are templates
+that will be shown to the user when they click the link in their threeid add
+email , either showing them a success or failure page (assuming a redirect URL
+is not configured).
+
+
+Synapse will expect these files to exist inside the configured template
+directory. To view the default templates, see `synapse/res/templates
 <https://github.com/matrix-org/synapse/tree/master/synapse/res/templates>`_.
 
+SMS
+---
+
+Following upgrade, the only way to maintain the ability to register via a phone
+number will be to continue to delegate SMS delivery via the matrix.org and
```

Comment:
well, that's not quite true. You can do it yourself with your own IS if that's a thing you want to do, but we're not going to help you get that configured.